pages/curriculum/[lessonSlug]/mentor/index.tsx

```diff
@@ -0,0 +1,204 @@
+import React, { useMemo, useState } from 'react'
+import {
+  AddExerciseMutation,
+  GetAppProps,
+  Module,
+  useAddExerciseMutation,
+  withGetApp
+} from '../../../../graphql'
+import { useRouter } from 'next/router'
+import { AdminLayout } from '../../../../components/admin/AdminLayout'
+import { DropdownMenu } from '../../../../components/DropdownMenu'
+import { FormCard, MD_INPUT } from '../../../../components/FormCard'
+import { formChange } from '../../../../helpers/formChange'
+import { exercisesValidation } from '../../../../helpers/formValidation'
+import ExercisePreview from '../../../../components/ExercisePreview'
+import styles from '../../../../scss/mentorPage.module.scss'
+import { get } from 'lodash'
+import { ApolloError } from '@apollo/client'
+import QueryInfo from '../../../../components/QueryInfo'
+import { errorCheckAllFields } from '../../../../helpers/admin/adminHelpers'
+import * as Sentry from '@sentry/nextjs'
+
+type HeaderProps<T> = {
+  lesson: T
+  addExerciseData?: AddExerciseMutation | null
+  loading: boolean
+  error?: ApolloError
+  setModule: (v: null | Omit<Module, 'author' | 'lesson'>) => void
+}
+const Header = <T,>({
```

Comment:
Should it be `<T>` instead of `<T,>`?

Reply:
`<T>` is treated as an HTML element. Adding the `,` make TS understands it's not an HTML element.